.bazelignore

```diff
@@ -0,0 +1 @@
+examples/helloworld
```

Comment:
of course if you bazelignore the nested folder, you can't use it as sources for a test.
In my #337 I did what we do in rules_nodejs which is tell bazel that `examples/helloworld/BUILD` is just a regular file and does not indicate a Bazel package.

Reply:
yeh that's better. will close in favour of #337